shared/constants/metametrics.ts

```diff
@@ -550,6 +550,7 @@ export enum MetaMetricsEventName {
   PortfolioLinkClicked = 'Portfolio Link Clicked',
   ProviderMethodCalled = 'Provider Method Called',
   PublicAddressCopied = 'Public Address Copied',
+  QuoteError = 'Quote Error',
```

Comment:
Out of curiosity, were these new event properties confirmed through the metrics team / @worldlyjohn

Reply:
Yup, we checked it with @worldlyjohn